src/MAUI/Maui.Samples/SettingsPage.xaml

```diff
@@ -90,10 +86,10 @@
                                                      HeightRequest="25"
                                                      IsEnabled="{x:OnPlatform Android=True,
```

Comment:
We don't need OnPlatform if we do the same value on each platform.
It should be enabled and visible everywhere by default, we can remove the setters entirely.